src/Sylius/Component/Grid/Filter/EntityFilter.php

```diff
@@ -29,8 +29,16 @@ public function apply(DataSourceInterface $dataSource, $name, $data, array $opti
         }
 
         $field = isset($options['field']) ? $options['field'] : $name;
+        $fields = isset($options['fields']) ? $options['fields'] : [$field];
```

Comment:
Then some sanity check that allows only for one of them to be set (`isset($options['field']) xor isset($options['fields']) === true`) would be nice, as setting them both would result in unexpected behaviour.